tsql/tsql.g4

```diff
@@ -1227,131 +1228,407 @@ id
 
 simple_id
     : ID
-    | ABSOLUTE
+    | key_words
```

Comment:
What reason to extract `key_words` to separated rule? It is not used anywhere else.